### PR TITLE
main/fakeroot: upgrade to 1.23

### PR DESCRIPTION
**Pull request by prspkt**

### CODE REVIEW

File: main/fakeroot/APKBUILD [hunk 1]
```diff
@@ -1,34 +1,32 @@
...
-url="http://fakeroot.alioth.debian.org/"
+license="GPL-3.0-or-later"
+url="https://packages.debian.org/fakeroot"
 depends=
 options="!checkroot"
 checkdepends="bash"
-makedepends_build="libtool autoconf automake po4a"
-makedepends_host="libcap-dev acl-dev linux-headers"
-makedepends="$makedepends_build $makedepends_host"
+makedepends="acl-dev autoconf automake libcap-dev libtool linux-headers po4a"
```
Comment (jirutka) [inline main/fakeroot/APKBUILD]: Why did you remove `makedepends_build` and `makedepends_host`? These are needed for cross-compilation.

Comment (prspkt) [reply main/fakeroot/APKBUILD]: @jirutka Apologies, i didn't know about that.
Reverted.

File: main/fakeroot/APKBUILD [hunk 2]
```diff
@@ -13,8 +13,7 @@ makedepends_build="libtool autoconf automake po4a"
 makedepends_host="libcap-dev acl-dev linux-headers"
 makedepends="$makedepends_build $makedepends_host"
 subpackages="$pkgname-doc"
-source="http://ftp.debian.org/debian/pool/main/f/$pkgname/${pkgname}_${pkgver}.orig.tar.bz2
-	busybox-getopt.patch
```
Comment (ncopa) [inline main/fakeroot/APKBUILD]: you no longer apply busybox-getopt.patch but the patch was not removed. Has the patch been applied upstream? Is the handling of args with spaces fixed upstream? (see https://bugs.alpinelinux.org/issues/3500)

Comment (prspkt) [reply main/fakeroot/APKBUILD]: `busybox-getopt.patch` removed. `fakeroot 1.23` runs as expected without util-linux.
Tested and confirmed against the script provided [here](https://bugs.alpinelinux.org/issues/3500).

File: main/fakeroot/APKBUILD [hunk 3]
```diff
@@ -4,30 +4,31 @@ pkgver=1.23
...
+url="https://packages.debian.org/fakeroot"
 depends=
 options="!checkroot"
 checkdepends="bash"
 makedepends_build="libtool autoconf automake po4a"
 makedepends_host="libcap-dev acl-dev linux-headers"
 makedepends="$makedepends_build $makedepends_host"
 subpackages="$pkgname-doc"
-source="http://ftp.debian.org/debian/pool/main/f/$pkgname/${pkgname}_${pkgver}.orig.tar.xz
+source="http://ftp.debian.org/debian/pool/main/f/fakeroot/${pkgname}_${pkgver}.orig.tar.xz
```
Comment (ncopa) [inline main/fakeroot/APKBUILD]: IMHO, this is a useful change, which makes it easier to copy paste the url to look for new versions. :+1:

File: main/fakeroot/APKBUILD [hunk 4]
```diff
@@ -4,30 +4,31 @@ pkgver=1.23
...
 	fakeroot-no-ldlibrarypath.patch
 	xstatjunk.patch
 	"
+builddir="$srcdir"/$pkgname-$pkgver
 
 check() {
-	cd $startdir/src/$pkgname-$pkgver
-	/bin/bash -c 'make check'
+	cd "$builddir"
+	make check
```
Comment (ncopa) [inline main/fakeroot/APKBUILD]: :+1:

File: main/fakeroot/APKBUILD [hunk 5]
```diff
@@ -4,30 +4,31 @@ pkgver=1.23
...
 }
 
 build() {
-	cd $startdir/src/$pkgname-$pkgver
+	cd "$builddir"
```
Comment (ncopa) [inline main/fakeroot/APKBUILD]: :+1:

File: main/fakeroot/APKBUILD [hunk 6]
```diff
@@ -48,7 +49,7 @@ build() {
 }
 
 package() {
-	cd "$startdir/src/$pkgname-$pkgver"
+	cd "$builddir"
```
Comment (ncopa) [inline main/fakeroot/APKBUILD]: :+1:
